api/v1alpha1/nnf_datamovementmanager_types.go

```diff
@@ -50,6 +51,10 @@ type NnfDataMovementManagerSpec struct {
 	// manages the per node data movement operations.
 	Template corev1.PodTemplateSpec `json:"template"`
 
+	// UpdateStrategy defines the UpdateStrategy that is used for the basis of the worker Daemon Set
+	// that manages the per node data movement operations.
+	UpdateStrategy appsv1.DaemonSetUpdateStrategy `json:"updateStrategy"`
```

Comment:
Does it need to be required?  Can it be omitempty?

Reply:
Never mind, I see that sos has it covered always.

Reply:
I originally had it as optional but then changed my mind to just make it a requirement. We basically never want it to use the default of 1 anyway, so I think expecting this field is the way to go.